src/client/datascience/types.ts

```diff
@@ -844,7 +845,7 @@ export type WebViewViewChangeEventArgs = { current: WebViewViewState; previous:
 export type GetServerOptions = {
     getOnly?: boolean;
     ui: IDisplayOptions;
-    localOnly?: boolean;
+    localOnly: boolean;
```

Comment:
Being explicy is better, we pass this around into `ConnectNotebookProviderOptions`, 
I found a lot of these and removed most, this is another one where I think its not great, as we have no idea where these are set at all, if you search through the code the property is never initialized becase we take one type & cast into another ..